app/assets/javascripts/editor.js

```diff
@@ -85,45 +85,50 @@ function snippet_changed(item){
   $('#association').toggle(!checked);
 }
 
-function create_editor(item) {
-  item.parent().prepend("<div id='editor1'></div>");
-  item.hide();
-  Editor = ace.edit("editor1");
+function create_editor() {
+  var editorId = "editor" + Math.floor((Math.random() * 100) + 1),
```

Comment:
On Wed, Aug 5, 2015 at 7:14 PM, Walden Raines <notifications@github.com>
wrote:

I'd rewrite this whole thing in AngularJS if I thought it would have a
> chance to be merged. Then we wouldn't need to add this unique ID nonsense
> and it would be tested. But alas.
>

What would it mean to write it in Angular? /me personally don't object
better ways to solve problems? Would it require the entire bastion plugin
to support it?